tools/protoc-plugin/src/test/proto/spine/tools/protoc/msg/test_events.proto

```diff
@@ -25,12 +25,12 @@
  */
 syntax = "proto3";
 
-package spine.tools.protoc.iface;
+package spine.tools.protoc.msg;
 
 import "spine/options.proto";
 
 option (type_url_prefix) = "type.spine.io";
-option java_package = "io.spine.tools.protoc.plugin.iface";
+option java_package = "io.spine.tools.protoc.plugin.message.tests";
 option java_outer_classname = "TestEventsProto";
 option java_multiple_files = true;
```

Comment:
I am not able to comment on `test_inner_messages.proto ` file, as it was just renamed and moved without changes in it. But it probably requires the updates in its Proto package name — since it was moved into another place of hierarchy.